cli/admin/src/test/java/io/pravega/cli/admin/segmentstore/StorageCommandsTest.java

```diff
@@ -143,6 +143,53 @@ public void testListChunksCommand() throws Exception {
         Assert.assertTrue(commandResult.contains("List of chunks for _system/containers/metadata_0"));
     }
 
+    @Test
+    public void testlistChunksWithFileSystemStorage() throws Exception {
+        int instanceId = 0;
+        int bookieCount = 3;
+        int containerCount = 1;
+
+        val baseDir = Files.createTempDirectory("testListChunks").toFile().getAbsoluteFile();
+        val storageConfig = FileSystemStorageConfig
+                .builder()
+                .with(FileSystemStorageConfig.ROOT, baseDir.getAbsolutePath())
+                .build();
+
+        this.storageFactory = new FileSystemSimpleStorageFactory(ChunkedSegmentStorageConfig.DEFAULT_CONFIG, storageConfig, executorService());
+        log.info("Created a long term storage.");
+
+        @Cleanup
+        LocalServiceStarter.PravegaRunner pravegaRunner = new LocalServiceStarter.PravegaRunner(bookieCount, containerCount);
+        pravegaRunner.startBookKeeperRunner(instanceId++);
+        this.factory = new BookKeeperLogFactory(pravegaRunner.getBookKeeperRunner().getBkConfig().get(), pravegaRunner.getBookKeeperRunner().getZkClient().get(),
+                executorService());
+        pravegaRunner.startControllerAndSegmentStore(this.storageFactory, this.factory, true);
+        String streamName = "testListChunksCommand";
+
+        TestUtils.createScopeStream(pravegaRunner.getControllerRunner().getController(), SCOPE, streamName, config);
+        try (val clientRunner = new TestUtils.ClientRunner(pravegaRunner.getControllerRunner(), SCOPE)) {
+            // Write events to the streams.
+            TestUtils.writeEvents(streamName, clientRunner.getClientFactory());
+        }
+
+        ServiceBuilder.ComponentSetup componentSetup = new ServiceBuilder.ComponentSetup(pravegaRunner.getSegmentStoreRunner().getServiceBuilder());
+        for (int containerId = 0; containerId < containerCount; containerId++) {
+            componentSetup.getContainerRegistry().getContainer(containerId).flushToStorage(TIMEOUT).join();
+        }
+
+        STATE.set(new AdminCommandState());
```

Comment:
This is not safe and not allowed. 
`STATE` is a static variable, we don't allow mutable statics. The reason is that tests are executed in parallel, so if two tests use the same static variable they will cause each other to fail. Please replace this with a local variable or a non-static member variable.

Reply:
The other two tests in this file still have the problem